### ISSUE TITLE
Stripe donations not working

### ISSUE THREAD

**Issue by luislavena** [syncthing/old-web#46]
Unsure if this is the right place to post this (since happened on the website, thought was worth report to the website codebase) :wink: 

Tried give a donation using Stripe, but received an error after submitting:

![screenshot from 2016-06-03 09-45-55](https://cloud.githubusercontent.com/assets/4182/15778982/0782adce-2970-11e6-9e4a-a3610c6a5dd0.png)

To my bad, had the network panel closed, so don't know precisely where the error is coming from.

Thank you.

/cc @calmh 

:heart: :heart: :heart:

**Comment by calmh** [comment]
Thanks for trying it, and for reporting the issue! From the logs on our side, it looks like an error reported by the bank so I'm guessing temporary communication issue or similar. Doesn't look like a technical problem on our side at least. (Despite our error message admitting the fault on our side by default. :)

**Comment by luislavena** [comment]
Thank you for the quick response.

I tried once again and got the same error. Funny thing is that I've used Stripe checkout in the past without issues (RubyTogether as example), wonder if the bank message is `card_declined` in which case might need to reach out my bank.

Will try again later today and hopefully it works, using Syncthing and love it so much that wanted to give a small token back as appreciation.

Thank you once again for your response and last but not least, for creating Syncthing.

Cheers.
